src/python/python-proto/python_proto/api/graph_query/v1beta1/messages.py

```diff
@@ -0,0 +1,835 @@
+
+    def into_proto(self) -> proto.NoMatchWithUid:
+        msg = self.new_proto()
+        return msg
+
+
+@dataclass(frozen=True, slots=True)
+class QueryGraphWithUidRequest(SerDe[proto.QueryGraphWithUidRequest]):
+    tenant_id: Uuid
+    node_uid: Uid
+    graph_query: GraphQuery
+    _proto_cls = proto.QueryGraphWithUidRequest
+
+    @classmethod
+    def from_proto(
+        cls, proto: proto.QueryGraphWithUidRequest
+    ) -> QueryGraphWithUidRequest:
+        return QueryGraphWithUidRequest(
+            tenant_id=Uuid.from_proto(proto.tenant_id),
+            node_uid=Uid.from_proto(proto.node_uid),
+            graph_query=GraphQuery.from_proto(proto.graph_query),
+        )
+
+    def into_proto(self) -> proto.QueryGraphWithUidRequest:
+        msg = proto.QueryGraphWithUidRequest()
+        msg.graph_query.CopyFrom(self.graph_query.into_proto())
+        msg.tenant_id.CopyFrom(self.tenant_id.into_proto())
+        msg.node_uid.CopyFrom(self.node_uid.into_proto())
+        return msg
+
+
+@dataclass(frozen=True, slots=True)
+class QueryGraphWithUidResponse(SerDe[proto.QueryGraphWithUidResponse]):
+    maybe_match: MaybeMatchWithUid
+    _proto_cls = proto.QueryGraphWithUidResponse
+
+    @classmethod
+    def from_proto(
+        cls,
+        proto: proto.QueryGraphWithUidResponse,
+    ) -> QueryGraphWithUidResponse:
+        return cls(
+            maybe_match=MaybeMatchWithUid.from_proto(proto.maybe_match),
+        )
+
+    def into_proto(self) -> proto.QueryGraphWithUidResponse:
+        msg = self.new_proto()
+        msg.maybe_match.CopyFrom(self.maybe_match.into_proto())
+        return msg
+
+
+@dataclass(frozen=True, slots=True)
+class QueryGraphFromUidRequest(SerDe[proto.QueryGraphFromUidRequest]):
+    tenant_id: Uuid
+    node_uid: Uid
+    graph_query: GraphQuery
+    _proto_cls = proto.QueryGraphFromUidRequest
+
+    @classmethod
+    def from_proto(
+        cls, proto: proto.QueryGraphFromUidRequest
+    ) -> QueryGraphFromUidRequest:
+        return cls(
+            tenant_id=Uuid.from_proto(proto.tenant_id),
+            node_uid=Uid.from_proto(proto.node_uid),
+            graph_query=GraphQuery.from_proto(proto.graph_query),
+        )
+
+    def into_proto(self) -> proto.QueryGraphFromUidRequest:
+        msg = proto.QueryGraphFromUidRequest()
+        msg.graph_query.CopyFrom(self.graph_query.into_proto())
+        msg.tenant_id.CopyFrom(self.tenant_id.into_proto())
+        msg.node_uid.CopyFrom(self.node_uid.into_proto())
+        return msg
+
+
+MaybeMatchWithUidInner = MatchedGraphWithUid | NoMatchWithUid
+
+
+@dataclass(frozen=True, slots=True)
+class MaybeMatchWithUid(SerDe[proto.MaybeMatchWithUid]):
+    inner: MaybeMatchWithUidInner
+    _proto_cls = proto.MaybeMatchWithUid
+
+    @classmethod
+    def from_proto(
+        cls,
+        proto_value: proto.MaybeMatchWithUid,
+    ) -> MaybeMatchWithUid:
+        field_name = proto_value.WhichOneof("inner")
+        assert field_name is not None
```

Comment:
'assert' is really weird in python, and I think it's best for a "this would be a bug" vs "this would be invalid" - instead I'd check and raise an exception